### PR TITLE
Update definitions & add tests for chrome.sessions api in @types/chrome

### PR DESCRIPTION
**Pull request by kishtra**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If adding a new definition:

- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)

### CODE REVIEW

**Review by fregante — COMMENTED** [review]
A couple of nits

File: types/chrome/index.d.ts [hunk 1]
```diff
@@ -7702,14 +7712,27 @@ declare namespace chrome.sessions {
      * Parameter devices: The list of sessions.Device objects for each synced session, sorted in order from device with most recently modified session to device with least recently modified session. tabs.Tab objects are sorted by recency in the windows.Window of the sessions.Session objects.
      */
     export function getDevices(callback: (devices: Device[]) => void): void;
+    /**
+     * Reopens a windows.Window or tabs.Tab.
+     * @param sessionId Optional.
+     * The windows.Window.sessionId, or tabs.Tab.sessionId to restore. If this parameter is not specified, the most recently closed session is restored.
+     * @return The `restore` method provides its result via callback or returned as a `Promise` (MV3 only).
```
Comment (fregante) [inline types/chrome/index.d.ts:7719]: I'd avoid mentioning the name again. Also this JSDoc is specific to promises so there's no need to mention callbacks. 

```suggestion
     * @return A promise that resolves with the Session data (MV3 only).
```

This also applies to the other method documented here.

Comment (kishtra) [reply types/chrome/index.d.ts:7746]: So, from what I found, out of 242 promise based definitions 212 use the first description of returned value (#53120) while 7 use the second description.
While I personally prefer your succint take I am not sure if we want to ensure consistency with the previous descriptions, update to new or change them on the fly.

**Review by fregante — APPROVED** [review]

**Comment by fregante** [comment]
Ready to merge

**Comment by fregante** [comment]
Ready to merge

**Pull request merged**